.github/workflows/poetry.yml

```diff
@@ -29,4 +29,4 @@ jobs:
       - name: Poetry run pytest
         run: ~/.poetry/bin/poetry run pytest
       - name: Poetry run black check
-        run: ~/.poetry/bin/poetry run black . --check
+        run: ~/.poetry/bin/poetry run black . --check --diff
```

Comment:
what does this do?

Reply:
During CI checks, this flag causes the check to print out the formatting changes that would happen for any unformatted files:

Here's an example
https://github.com/kitzeslab/opensoundscape/runs/1989043264